src/Gateway.php

```diff
@@ -1342,51 +1342,51 @@ public function update_payment_from_mollie_payment( Payment $payment, MolliePaym
 				$bank_transfer_recipient_details->set_bank_account( $bank_details );
 			}
 
-			if ( isset( $mollie_payment_details->bankName ) ) {
+			if ( $mollie_payment_details->has_property( 'bankName' ) ) {
 				/**
 				 * Set `bankName` as bank details name, as result "Stichting Mollie Payments"
 				 * is not the name of a bank, but the account holder name.
 				 */
-				$bank_details->set_name( $mollie_payment_details->bankName );
+				$bank_details->set_name( $mollie_payment_details->get_property( 'bankName' ) );
 			}
 
-			if ( isset( $mollie_payment_details->bankAccount ) ) {
-				$bank_details->set_iban( $mollie_payment_details->bankAccount );
+			if ( $mollie_payment_details->has_property( 'bankAccount' ) ) {
+				$bank_details->set_iban( $mollie_payment_details->get_property( 'bankAccount' ) );
 			}
 
-			if ( isset( $mollie_payment_details->bankBic ) ) {
-				$bank_details->set_bic( $mollie_payment_details->bankBic );
+			if ( $mollie_payment_details->has_property( 'bankBic' ) ) {
+				$bank_details->set_bic( $mollie_payment_details->get_property( 'bankBic' ) );
 			}
 
-			if ( isset( $mollie_payment_details->transferReference ) ) {
-				$bank_transfer_recipient_details->set_reference( $mollie_payment_details->transferReference );
+			if ( $mollie_payment_details->has_property( 'transferReference' ) ) {
+				$bank_transfer_recipient_details->set_reference( $mollie_payment_details->get_property( 'transferReference' ) );
 			}
 
 			/*
-			 * Failure reason.
-			 */
+			* Failure reason.
```

Comment:
Remove comment?

Reply:
All these parts have comments with what they're about:

- Consumer bank details.
- Bank transfer recipient details.
- Failure reason.

If we remove the failure reason comment, then we might as well remove the others too?